### ISSUE TITLE
BUG: "Refresh Table" button does not display the added primary or foreign key

### ISSUE THREAD

**Issue by lucafornerone** [beekeeper-studio/beekeeper-studio#2500]
**Describe the bug**
If I have the table view open in one tab and I create a foreign key (on the same table) through SQL script in another tab, clicking the "Refresh Table" button does not display the foreign key icon on the column that was just updated. Same for primary key.

**To Reproduce**
1. Connect to a database with at least one table
2. In the left sidebar, right click on a table
3. Click on "View Data"
4. Open a new tab
5. Add a foreign key through SQL script on the same table opened
6. Return to the previous tab
7. Click on "Refresh Table" in the bottom right corner
8. The column that was just updated **not display** the foreign key icon

**Expected behavior**
The column that was just updated **display** the foreign key icon

**Version Information (please complete the following information):**
 - OS: Windows 11
 - App Version: 4.6.8
 - Database type and version: Posgresql 14.10

**Comment by not-night-but** [comment]
@lucafornerone Yeah this is an issue, we probably just don't refresh the schema/keys when we refresh the data

**Comment by SMV-21** [comment]
Hi, I’d like to work on this issue. Could it be assigned to me?
